crates/biome_js_analyze/tests/specs/nursery/useTrimStartEnd/valid.js

```diff
@@ -0,0 +1,26 @@
+const foo = bar.trimStart();
+const foo = bar.trimEnd();
+bar.trimStart?.()
+bar['trimStart']()
```

Comment:
- `bar["trimStart"]()`
- ``bar[`trimStart`]()``

The counterparts should also be added to the invalid test cases.

Reply:
Sorry for not being clear, I meant adding these two cases here, plus `bar['trimStart']()`. Not deleting them.

And also adding something like `foo['trimLeft']()`, `foo["trimLeft"]()` and ``foo[`trimLeft`]()`` to the invalid cases.

Reply:
Sorry, I've fixed this.